README.md

```diff
@@ -237,8 +236,8 @@ mypy
 
 If you modify any database models (`models.py`), you'll need to run the following to update the database:
 ```shell
-python manage.py makemigrations
-python manage.py migrate
+poetry run python manage.py makemigrations
+poetry run python manage.py migrate
```

Comment:
Shouldn't this be
```suggestion
poetry run manage.py makemigrations
poetry run manage.py migrate
```

Reply:
no, per their docs you run stuff normally with `python` but you just prefix commands with `poetry run`